packages/react-accordion/Spec.md

```diff
@@ -0,0 +1,299 @@
+# Accordion
+
+## Background
+
+### Definition
+
+This spec defines the default function of an `Accordion` as a vertically stacked set of interactive panels that each contain a title and content snippet.
+
+## Prior art
+
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([research proposal](https://github.com/WICG/open-ui/pull/263))
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
+
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
+
+There's no comparison to be done between the two libraries, since v8 hasn't implemented this component.
+
+## Issues
+
+- #16553
+- #16452
+- #12953
+- #12238
+
+## API
+
+The `Accordion` should implement a `children` based API as is the standard across all the surveyed alternatives as a part of Open UI research in [Prior Art](#prior-art). The component will leverage the use of `context` in the interaction and data flows of child components.
+
+Sample usages will be given in the following section of this document [Sample code](#sample-code)
+
+### Accordion
+
+The root level component serves context and common API between all children.
+
+| Prop name          | Type                    | Default Value | Details                                                      |
+| ------------------ | ----------------------- | ------------- | ------------------------------------------------------------ |
+| expand (?)         | boolean                 | false         | Allows multiple panels to be expanded                        |
+| collapse (?)       | boolean                 | false         | Allows multiple panels to be collapsed                       |
```

Comment:
Would this be confusing for consumers ? that there is an implicit relationship between `collapse` and `expand` which don't really describe the relationship very explicitly

Reply:
This might also be a bit too much detail before some initial implementation, but do you envision this component being totally uncontrolled ? How would it work from a controlled perspective

Reply:
IMPO every component in a Design System should support both Uncontrolled and Controlled patterns, for sure I still need to include more details on that with some `default` props.

Reply:
In that case as we discussed offline, we need to define perhaps that these props would be for uncontrolled cases only, since it might not make sense to control opening of individual items and have these apply